KenticoCloud.Delivery/Extensions/HttpRequestHeadersExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Diagnostics;
+using System.Net.Http.Headers;
+using System.Reflection;
+
+namespace KenticoCloud.Delivery.Extensions
+{
+    internal static class HttpRequestHeadersExtensions
+    {
+        private const string SdkTrackingHeaderName = "X-KC-SDKID";
+        private const string WaitForLoadingNewContentHeaderName = "X-KC-Wait-For-Loading-New-Content";
+
+        private const string PackageRepositoryHost = "nuget.org";
+        private const string SdkPackageId = "KenticoCloud.Delivery";

```

Comment:
i think this can be safely extracted from assembly information as well (assembly.GetName().Name)